src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -251,6 +251,53 @@ describe('Disables proctoring provider options', () => {
   });
 });
 
+describe('Hides fields based on user permissions', () => {
+  beforeEach(async () => {
+    auth.getAuthenticatedHttpClient = jest.fn(() => ({
+      get: async () => ({
+        data: {
+          proctored_exam_settings: {
+            enable_proctored_exams: true,
+            allow_proctoring_opt_out: false,
+            proctoring_provider: 'mockproc',
+            proctoring_escalation_email: 'test@example.com',
+            create_zendesk_tickets: true,
+          },
+          available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
+          course_start_date: '2070-01-01T00:00:00Z',
+        },
+        catch: () => {},
+      }),
+    }));
+  });
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  function mockAuthentication(isAdmin) {
+    auth.getAuthenticatedUser = jest.fn(() => ({ userId: 3, administrator: isAdmin }));
+  }
+
+  it('hides opting out and zendesk tickets for non edX staff', async () => {
+    mockAuthentication(false);
+    await act(async () => render(<ProctoredExamSettings {...defaultProps} />));
+    const allowOptingOut = screen.queryByTestId('allowOptingOutYes');
```

Comment:
#NIT These 4 lines can be compacted into two lines. Just do `expect(screen.queryByTestId('allowOptingOutYes')).toBeNull();`